README.md

```diff
@@ -28,6 +41,14 @@ The application accepts the following parameters:
 
 ### Docker
 
+*Prerequisites*
```

Comment:
Instead of repeating myself, I would prefer just adding `git submodule update --init --recursive` before the build command below.

Reply:
I do not like adding commands without explanations or context.